src/bp/sdk/botpress.d.ts

```diff
@@ -1719,23 +1719,26 @@ declare module 'botpress/sdk' {
     actions: ActionButton[]
   }
 
+  export enum ButtonAction {
```

Comment:
Dupe ?

Reply:
We define the enum twice like other enums in the sdk. For example `NodeActionType`